deliver/lib/deliver/setup.rb

```diff
@@ -60,9 +60,13 @@ def deliverfile_path
       end
     end
 
-    def generate_metadata_files(app, version, path)
+    def generate_metadata_files(app, version, path, options)
       # App info localizations
-      app_info = app.fetch_live_app_info || app.fetch_edit_app_info
+      if options[:use_live_version]
+        app_info = app.fetch_live_app_info
```

Comment:
I think we want to display an error if `app.fetch_live_app_info` returns `nil` 🤔  Thoughts?

Reply:
I couldn't agree more - nice thought! Otherwise it would just throw a mysterious crash below just like it used to 🙈  I'll update it 🙌